app/src/main/java/org/jellyfin/androidtv/ui/browsing/EnhancedBrowseFragment.java

```diff
@@ -65,7 +66,7 @@
 import static org.koin.java.KoinJavaComponent.inject;
 
 public class EnhancedBrowseFragment extends Fragment implements IRowLoader {
-    BaseActivity mActivity;
+    FragmentActivity mActivity;
```

Comment:
Do you want to add a scope here to appease Codacy?

Reply:
Added it to all properties